spoutbreeze-manager/build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
 	implementation 'org.springframework.session:spring-session-core'
     implementation 'org.springframework.boot:spring-boot-starter-amqp'
   	implementation 'org.spoutbreeze:spoutbreeze-commons:0.0.1-SNAPSHOT'
+	implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.11.1'
```

Comment:
To parse the json message obtained from the queue

Reply:
jackson library is fine